lib/v2/medium/list.js

```diff
@@ -0,0 +1,32 @@
+const config = require('@/config').value;
+
+const parseArticle = require('./parse-article.js');
+const { getUserCatalogMainContentQuery } = require('./graphql.js');
+
+module.exports = async (ctx) => {
+    const user = String(ctx.params.user);
+    const catalogId = String(ctx.params.catalogId);
+
+    const cookie = config.medium.cookies[user];
+
+    const catalog = await getUserCatalogMainContentQuery(user, catalogId, cookie);
```

Comment:
It would be better to throw an exception if the cookie is not set before any network requests.